proxy/static/index.html

```diff
@@ -20,7 +20,6 @@
         <div class="btn-group dropdown pull-right">
 
           <button id="language-switcher" type="button" class="btn btn-dark bg-dark dropdown-toggle btn-block" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            en_US
```

Comment:
Instead of deleting `en_US`, should we replace it with "Language"? Or rather, a localised string like `__MSG_language__`? Hopefully, this will make it clear that the drop down menu is a language switcher.

Reply:
I removed it because no matter what we put there, it's populated automatically on page load here: https://github.com/cohosh/snowflake/pull/9/files#diff-f5cef12a6513451c2d2fffdcef07aaa8R68